spec.html

```diff
@@ -9213,7 +9214,6 @@ <h1>[[Construct]] ( _argumentsList_, _newTarget_ )</h1>
         1. Let _target_ be _O_.[[ProxyTarget]].
         1. Let _trap_ be ? GetMethod(_handler_, `"construct"`).
         1. If _trap_ is *undefined*, then
-          1. Assert: IsConstructor(_target_) is *true*.
```

Comment:
`Construct` on the next step asserts this.

Reply:
I think the assertion here adds clarity. Can we leave it?

Reply:
I've hoisted the assertion: this internal method is never created (and invoked) on a proxy if target is not a constructor.